input/tcp.go

```diff
@@ -85,21 +87,29 @@ func (s *TCP) Run(inch chan<- *baker.Data) error {
 	}
 	defer l.Close()
 
+	wg := sync.WaitGroup{}
 	for atomic.LoadInt64(&s.stop) == 0 {
 		l.SetDeadline(time.Now().Add(1 * time.Second))
 		conn, err := l.AcceptTCP()
 		if err != nil {
-			if opErr, ok := err.(*net.OpError); ok && opErr.Timeout() {
+			if errors.Is(err, os.ErrDeadlineExceeded) {
```

Comment:
asking: is this the same semantic as before?
I see `os.ErrDeadlineExceeded` represents a generic I/O timeout, but the doc for `opErr.Timeout()` is quite... well, empty 😅

Reply:
yes it's the same semantic as before. `opErr.Timeout` is very low level and was the only way when this input has been created, in our case it's preferred to use `os.ErrDeadlineExceeded` since it's the error we're looking for